cluster-autoscaler/go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0
 	github.com/Azure/go-autorest/autorest/to v0.2.0
 	github.com/aws/aws-sdk-go v1.35.24
+	github.com/brightbox/gobrightbox v0.5.7
```

Comment:
We do not accept any additional dependencies that are not in kubernetes/kubernetes. Please in-line your deps as in other cloud providers do.

Reply:
In-lined the dependencies as requested.